deps/jerry/tools/snap.js

```diff
@@ -0,0 +1,122 @@
+var fs = require('fs');
```

Comment:
Strict mode is required.

Reply:
Where should this file be? The CPU profiling tool are https://github.com/Rokid/ShadowNode/tree/master/tools/profiler.

Reply:
I would better place CPU and heap profiler into jerry source tree, because it is not related to shadow-node. The CPU profiler scripts will also be moved in other PR.